lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -86,6 +98,100 @@ sub execute {
     return 1;
 }
 
+sub get_bam_from_model {
+    my $self = shift;
+    my $model = Genome::Model->get(id => $self->model);
```

Comment:
(In the event that the `model` parameter is not changed as suggested, it is important to manually verify we really got a model here.)